modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -133,7 +138,20 @@ status: {}
 <1> Remove these sections completely.
 +
 If you do so, you must add ingress DNS records manually in a later step.
```

Comment:
Is that a step that is included in these docs? If so I would think linking it here would be helpful.
If not, please let us know that would be done post UPI deployment.

Reply:
Agreed! I'll provide the link here.